docs/building-images.md

```diff
@@ -331,7 +332,8 @@ The following sections further describe optional directories that may be include
     in combination with the manifests section in the definition file. All files in this directory will be parsed and
     the container images that they reference will be downloaded and served in an embedded artefact registry.
   * `helm`
-    * `values` Contains [Helm values files](https://helm.sh/docs/chart_template_guide/values_files/). Helm charts that require specified values must have a values file.
+    * `values` - Contains [Helm values files](https://helm.sh/docs/chart_template_guide/values_files/). Helm charts that require specified values must have a values file.
+    * `certs` - Contains CA files for TLS verification. Helm repositories and registries with untrusted certificates must either use a CA File or enable `skipTLSVerify` for HTTPS-enabled servers.
```

Comment:
I'm not sure whether the description is the right one here and would love a second pair of eyes on it. LGTM otherwise.